docs/cql/functions.rst

```diff
@@ -33,13 +32,16 @@ CQL supports two main categories of functions:
 - The :ref:`aggregate functions <aggregate-functions>`, which are used to aggregate multiple rows of results from a
   ``SELECT`` statement.
 
-.. In both cases, CQL provides a number of native "hard-coded" functions as well as the ability to create new user-defined
-.. functions.
+In both cases, CQL provides a number of native "hard-coded" functions as well as the ability to create new user-defined
+functions.
+
+.. note:: Although user-defined functions are sandboxed, protecting the system from a "rogue" function, user-defined functions are disabled by default for extra security.
+. See the ``enable_user_defined_functions``
+   in ``scylla.yaml`` to enable them.
 
-.. .. note:: By default, the use of user-defined functions is disabled by default for security concerns (even when
-..    enabled, the execution of user-defined functions is sandboxed and a "rogue" function should not be allowed to do
-..    evil, but no sandbox is perfect so using user-defined functions is opt-in). See the ``enable_user_defined_functions``
-..    in ``scylla.yaml`` to enable them.
+   Additionally, user-defined functions are still experimental, and need to be explicitly enabled by adding ``udfs`` to the list of
+   ``experimental_features`` configuration option in ``scylla.yaml``, or turning on the ``experimental`` flag in order
+   to enable all experimental features, UDFs included. Use in test environment only!
```

Comment:
I think perhaps the phrase "Use in test environment only!" is a bit scary. I think we should have a single page explaining in detail the Scylla concept of "experimental" (that it's not tested as well, backward compatibility is not guaranteed in new versions, and upgrades are not guaranteed) - and then all places describing experimental features can link to such a page. But we don't need to do that now (or, maybe we already have such a page?) CC @tzach @annastuchlik

Reply:
Agree. "experimental" is scary enough.

Reply:
I'll add a link to https://docs.scylladb.com/stable/operating-scylla/admin.html#enable-specific-experimental-features.
It explains how to enable an experimental feature and will be extended with more details soon.